xarray_sentinel/sentinel1.py

```diff
@@ -248,6 +248,12 @@ def open_swath_dataset(
     for pol, data_path in measurement_paths.items():
         arr = rioxarray.open_rasterio(data_path, chunks=chunks)
         arr = arr.squeeze("band").drop_vars(["band", "spatial_ref"])
+        arr.coords.update(
```

Comment:
I think the official API is `.assign_coords()`: http://xarray.pydata.org/en/stable/generated/xarray.DataArray.assign_coords.html